README.md

```diff
@@ -42,6 +42,8 @@ cfn test
 cfn test -- -k contract_delete_update #to run a single test
 cfn test --enforce-timeout 60 #set the RL handler timeout to 60 seconds and CUD handler timeout to 120 seconds.
 cfn test --enforce-timeout 60 -- -k contract_delete_update # combine args
+cfn test --region "us-west-2" --partition "aws" --account-id "12345678" #allows users to set a valid region, account-id and partition.
```

Comment:
you shouldn't need to pass in the account id or partition. All of this should be available from the session we are using:

https://docs.aws.amazon.com/cli/latest/reference/sts/get-caller-identity.html

Reply:
So I had to [use the partition/region](https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-cloudformation/pull/4/files#diff-3fcebf9413b88490d8d6d7c17cb290cdR134) in my `AWS::CloudFormation::ResourceVersion` type, because I need to predict an Arn to comply with sync response requirement on Create. Can I extract this from the context I have available in the handler?

Reply:
Ideally we shouldn't require any additional permissions to run the contract tests other than the permissions needed to create/update/delete their resources. We could add default behavior to call the api, but we should allow people to pass in the account id as well